CONTRIBUTING.md

```diff
@@ -110,7 +110,7 @@ the main (upstream) repository:
 # Development Environment
 
 Please make sure to install the project requirements,
-see the [dependencies](./README.md#dependencies) section in top README.
+see the [dependencies](./README.rst#prerequisites) section in top README.
```

Comment:
In fact, it failed my local build. I guess we can't link a `rst` file from a `md` file with such syntax.